### PR TITLE
Setup the base of project based on FastApi

### PR DESCRIPTION
**Pull request by sumit-158**
### What
- Initializing the project based on FastApi 
- Also add docker-compose 
- updated readme

### Fixes bug(s)
- #1

### CODE REVIEW

**Comment by sumit-158** [comment]
@alexgarel  pytest is skipping with this message 
image

**Comment by sumit-158** [comment]
> @alexgarel pytest is skipping with this message 

Sloved!

**Review by alexgarel — COMMENTED** [review]
TODO:
- move test_main.py to its own directory
- change the current description for it would only solve issue #1 yet (and it's better to have separate PRs if possible)
- update README to explain how to setup the project and run the tests

File: README.md [hunk 1]
```diff
@@ -1,2 +1,43 @@
 # facets-knowledge-panels
 Providing knowledge panels for a particular open food fact facet (category, brand, etc...)
+
+## why facets-knowledge-panels?
+
+- Apps don't have to add special code for each panel (e.g. looking at the array of ingredients and additives, the values of nutrients etc.)
+- Apps don't need to load and update taxonomies to interpret the returned data
+- Translations are managed by the server, and we can use Crowdin to crowdsource them in many languages
+- If desired, apps can decide to use new product attributes added on the server without requiring app changes
```
Comment (alexgarel) [inline README.md:9]: This is more the why of knowledge panels. (not specific to facets)

But the examples that are given are not consistent with facet.

I would rather say.:
```suggestion
Provides applications with a set of informative or actionable items which are contextual to a specific facet.

The primary goal is to allow high level contributions by users that maybe interested in a particular subset of the database, like a particular food category or a brand, a location, etc...

We reuse the knowledge panel format, which remove the need for the application to change as the server side code change, or to implement specific business logic.

```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,43 @@
...
+uvicorn main:app --reload
+```
+- Checkout your local host `http://127.0.0.1:8000/` with the endpoints
+
+## Testing
+
+- Checkout `tests` directory
+```
+pytest 
+```
```
Comment (alexgarel) [inline README.md:42]: This is true if you use "virtual env" if you use docker you have to launch with docker-compose.
````suggestion
### docker setup
```
docker-compose run --rm facets-api pytest tests
```

### virtual env setup
- Checkout `tests` directory
```
pytest 
```
````

**Review by alexgarel — CHANGES_REQUESTED** [review]
Hi @sumit-158 , good job.

Small changes requested.
+ remember that this only fixes first bug. Please remove the other two from your comment.

**Comment by sumit-158** [comment]
> Small changes requested.
> 
> * remember that this only fixes first bug. Please remove the other two from your comment.

Done , I hope everything is good now :)

**Comment by sumit-158** [comment]
> * remember that this only fixes first bug. Please remove the other two from your comment.

but, @alexgarel it also return hunger game KP as you shows in ticket #3

File: app/main.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+@app.get("/brand/president")
+
+def ansewer_questions_brand_president():
+    return {
+            "knowledge_panels": [
+                {
+                    "hunger-game": {
+                        "type": "hunger-game",
+                        "level": "questions",
```
Comment (alexgarel) [inline app/main.py:15]: I don't think this two values are ok, but it's not blocking right now :-)

**Review by alexgarel — APPROVED** [review]
Perfect @sumit-158  !

I installed and run locally :-) It worked like a charm.

**Pull request merged**